exporters/otlp/README.md

```diff
@@ -0,0 +1,83 @@
+# opentelemetry-exporters-otlp
+
+The `opentelemetry-exporters-otlp` gem provides an OTLP exporter for OpenTelemetry for Ruby. Using `opentelemetry-exporters-otlp`, an application can configure OpenTelemetry to export collected tracing data to [the OpenTelemetry Collector][opentelemetry-collector-home].
```

Comment:
Might be nice to make `OTLP` in `...provides an OTLP exporter...` a hyperlink to the repo for OTLP so anyone exploring this repo can find more information easily.